docs(Bank Reconciliation): Redesign (#230)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/accounts/bank-reconciliation.md

1. The Mapping that is done is stored in the Bank Document linked in the corresponding Bank Account. At the next upload the Mapping is taken from here but it is allowed to change it. The changed Mapping is updated in the Bank Document too.
 <img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/bank_configuration.png">

#### 3.2.2 Bank account synchronization

You can use Plaid (see [Plaid Integrations page](/docs/user/manual/en/erpnext_integration/plaid_integration)) to automatically synchronize your bank account with ERPNext. All your bank transactions will be automatically imported into ERPNext.

#### 3.2.3 Reconcile the Bank Statement

Once all your bank transactions are imported into ERPNext, you can reconcile them with your existing Vouchers. Go To
> Accounting > Bank Statement > Bank Reconciliation Tool

or simply search for 'Bank Reconciliation Tool' in the awesomebar.

1. Select your Company, Bank Account, Bank Statement Start and End Date.
1. Make sure that the Opening Balance from ERPNext Matches the Opening Balance of your Bank Statement.
1. Enter the Closing Balance of the Bank Statement.

> 1. Saving the Document will create the Tool.

Reviewer: @hasnain2808 Saving the Document will create the Tool.

What does this mean?

> 1. Saving the Document will show the matching Bank Transactions.

Reviewer: ```suggestion
1. Saving the document will show the matching bank transactions.
```

 <img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/bank_reconciliation_tool.png">

1. The Final goal of Bank Reconciliation is to make the Difference amount zero (Green) by either matching to an existing Voucher or creating a new voucher.

> 1. For All bank Transactions which are present in the Bank Statement but clearance date is not updated click on the Actions Button you want to Match/ Create Vouchers

Reviewer: recheck this sentence

> 1. For All bank Transactions which are present in the Bank Statement but clearance date is not updated click on the Actions Button to Match/ Create Vouchers

Reviewer: rephrase this... make it simpler

> 1. For All bank Transactions which are present in the Bank Statement but clearance date is not updated click on the Actions Button to Match/ Create Vouchers

Reviewer: ```suggestion
1. For all the bank transactions which are present in the Bank Statement but do not have a clearance date, click on the Actions Button to Match/ Create Vouchers
```

> 1. For Matching Choose 'Match Against Voucher' in 'Action', The Vouchers that ERPNext found related will be displayed. They will be ranked on the basis of maximum number of fields matched. You can match one or multiple Vouchers against the same Bank Transaction using the checkboxes.

Reviewer: ```suggestion
1. For matching, choose 'Match Against Voucher' in 'Action'. The vouchers that are related to this transaction will be displayed. They will be ranked on the basis of the maximum number of fields matched. You can match one or multiple vouchers against the same Bank Transaction using the checkboxes.
```

Reviewer: which checkboxes?

<img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/match_voucher.png">

> 1. To create a new voucher, choose Create Voucher in the Actions and then choose the Document Type. Fill the details that were not available in the Bank Transaction. Clicking on Submit will create the corresponding voucher and update it's clearance date.

Reviewer: ```suggestion
1. To create a new voucher, choose 'Create Voucher' in the 'Action' and then choose the document type. Fill in the details that were not available in the Bank Transaction. Clicking on Submit will create the corresponding voucher and update its clearance date.
```

 <img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/create_voucher.png">

> 1. It is also possible to update the Bank Transaction so that ERPNext can do a better job at finding related Vouchers. Choose Update Bank Transaction in Action, FIll the required details and click on Submit to Save the Bank Transaction.

Reviewer: It is also possible to update the Bank Transaction so that ERPNext can do a better job at finding related Vouchers. 

- reframe this sentence.

> 1. It is also possible to update the Bank Transaction so that ERPNext can do a better job at finding related Vouchers. Choose Update Bank Transaction in Action, FIll the required details and click on Submit to Save the Bank Transaction.

Reviewer: Choose 'Update Bank Transaction' in 'Action'. Fill the required details and click on Submit to save the Bank Transaction.

> 1. It is possible to update the Bank Transaction. Updating the Bank Transaction might help ERPNext in finding better matches. To Update a Bank transaction, choose 'Update Bank Transaction' in 'Action', fill the required details and click on Submit to save the Bank Transaction.

Reviewer: ```suggestion
1. It is also possible to update the Bank Transactions. Updating the Bank Transaction might help ERPNext in finding better matches. To Update a Bank transaction, choose 'Update Bank Transaction' in 'Action', fill in the required details, and click on Submit to save the Bank Transaction.
```
